From 73ca2d5f84c5728712c00135ff82a42ae3487b8c Mon Sep 17 00:00:00 2001
From: Seth Maurice-Brant <37915796+iSaluki@users.noreply.github.com>
Date: Fri, 17 Nov 2023 21:54:30 +0000
Subject: [PATCH 050/188] Change Fedora installation to the new official Helix
 package (#8762)

Remove COPR install in favour of official Helix package
---
 book/src/install.md | 3 ---
 1 file changed, 3 deletions(-)

--- a/book/src/install.md
+++ b/book/src/install.md
@@ -65,10 +65,7 @@ sudo apt install helix
 
 ### Fedora/RHEL
 
-Enable the `COPR` repository for Helix:
-
 ```sh
-sudo dnf copr enable varlad/helix
 sudo dnf install helix
 ```
 
